flask/json.py

```diff
@@ -242,11 +256,21 @@ def get_current_user():
         indent = 2
         separators = (', ', ': ')
 
+    if args and kwargs:
+        raise TypeError(
+            "jsonify() behavior undefined when passed both args and kwargs"
+        )
+    elif len(args) == 1: # single args are passed directly to dumps()
```

Comment:
Oh, I didn't know that. I've been meaning to setup a pep8 checker on my local workflow, just haven't gotten to it.